sylt-common/src/error.rs

```diff
@@ -417,6 +422,14 @@ impl fmt::Display for TypeError {
                 write!(f, "A tuple of length {} has no element {}", length, got)
             }
 
+            TypeError::TupleLengthMismatch { length, got } => {
+                write!(
+                    f,
+                    "Tuple lengths don't match '{}'!='{}'",
```

Comment:
```suggestion
                    "Tuple lengths don't match, expected {} but got {}",
```

Reply:
Expected and got doesn't really make sense, since they're just two tuples that don't share lengths. The naming is actually wrong.